examples/test_suite_1/test_suite_1.py

```diff
@@ -96,11 +96,7 @@ def setUp(self):
 
     @pykiso.retry_test_case(max_try=5, rerun_setup=True, rerun_teardown=True)
     def test_run(self):
-        """In this case the default test_run method is overridden and
-        instead of calling test_run from RemoteTest class the following
-        code is called.
-
-        Here, the test pass at the 3rd attempt out of 5. The setup and
+        """In this case the default test_run method is overridden andinstead of calling test_run from RemoteTest class the following code is called. Here, the test pass at the 3rd attempt out of 5. The setup and
```

Comment:
why breaking the format ?
if this was done by intention space is missing ->andinstead and ...

Reply:
The modification on test_suite_1 and dummy.yaml were not supposed to be pushed, my mistake.